---
title: "hw4"
author: "Jiahao Tian"
date: "2023-02-18"
output: pdf_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
library(ribd)
library(xtable)
```


# Chapter 7


# Qestion 2


```{r echo=FALSE, out.width = "75%", fig.align = "center"}
knitr::include_graphics("~/Desktop/234.png")
```


- 2/1 x 2/3 and 2/3 x 2/1 are possible. The both parents (3, 4) must carry an 2 allele because of genotype 2/2 for the child 5 and, second, that one parent must carry an 1 allele and the other carry an 3 allele because of the presence of an 1 allele in the child 6 and an 3 allele in the child 7. Neither the genotype 1/1 of the child 6 nor the genotype 3/3 of the child 7 is compatible with either of these two parental mating types. Hence, step B of the algorithm applied to this family {3, 4 , 5, 6 ,7} produces the genotype sets shown in the figure above.

- Now, consisting of individuals 1, 2, and 3. If indivedual 1 is assigned the genotype 1/3, individual 2 can be assigned the genotype 2/2. The mating type 1/3 x 2/2 then produces the child genotype 1/2 as one of its zygotes. Step B applied to the family {1, 2, 3}, but 1/1, 3/3 are not possible for individuals 1 and 2, therefore yields the situation shown in above figure. Step C amounts to repetition of step B for each nuclear family. In both cases, no new genotypes are eliminated and the algorithm stops.

- The pedigree in figure contains a brother x sister mating between individuals 3 and 4. Since 3 and 4 have identical genotype sets after the first application of step B to the nuclear family 3, 4, 5, 6, and 7, inclusion of individual 4 in the second nuclear family can add no new information to the exclusion process involving 1, 2, and 3. However, the two children, 3 and 4, must have either of the symmetric
mating types 1/2 x 3/2 or 3/2 x 1/2 in the context of their own family 3, 4, 5, 6, and 7. It follows that none of the two genotypes 1/1, 3/3 are possible for individuals 1 and 2 after the conclusion of the algorithm can form part of a compatible genotype for the whole pedigree. Thus, the algorithm is not always fully efficient in the presence of inbreeding loops.


# Quesiton 7


```{r echo=FALSE, out.width = "75%", fig.align = "center"}
knitr::include_graphics("~/Desktop/dd.png")
```


- Assume a recessive autosomal disease gene e (with alleles E and e) is linked to markers a, b, c, and d, respectively. 

- In order to produce affected kids numbers 3 and 5, both parents must be carriers of the Ee genotype. So there are different ways to combine the markers with the autosomal alleles E and e. For mom we can have (Ea/eb, ea/Eb), for dad we can have (Ec/ed, ec/Ed), etc.

- We already know that fetus 5 has the markers b/c, where b is 100% from mom and c is 100% from dad. This fetus must inherit recessive alleles from both parents in order to be affected. We end up with (be/ce).

- Because there are different ways to combine the markers with the autosomal alleles E and e for both parents. So first we assume (Ea/eb) and (Ec/ed) are the parental genotype, another ways are from recombination. See following table.



$$\begin{aligned}
 &\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;Given\;phase\;i,\;probability\;that\;fetus\;5\;is\\
  \hline
      & \;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;phase\;i\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;eb\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;Eb\\ 
  1 \;& Ea/eb \;\;\; (1-\theta)(1-\theta) \;\;\;\;\;\;(1-\theta)(1-\theta)\;\;\;\;\;\;\;\;\;\theta \cdot \theta\\
  2 \;& ea/Eb \;\;\;\;\;\;\;\;\;\;\;\theta \cdot \theta\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\theta \cdot \theta\;\;\;\;\;\;\;\;\;(1-\theta)(1-\theta)\\
  &\cdot\\
  &\cdot\\
  &\cdot\\
      & \;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;phase\;i\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;ec\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;Ec\\
  3 \;& Ec/ed \;\;\; (1-\theta)(1-\theta)\;\;\;\;\;\;\;\;\;\;\;\;\;\theta \cdot \theta\;\;\;\;\;\;\;\;\;\;(1-\theta)(1-\theta)\\ 
  4 \;& ec/Ed \;\;\;\;\;\;\;\;\;\;\;\theta \cdot \theta\;\;\;\;\;\;\;\;\;\;\;\;\;\;(1-\theta)(1-\theta)\;\;\;\;\;\;\;\;\;\theta \cdot \theta\\ 
  &\cdot\\
  &\cdot\\
  &\cdot\\
  \hline
\end{aligned}$$



$$\begin{aligned}
&From\;above\;table:\\
P(fetus\;5\;is\;"genotype") & = \sum_{i=1}^i P(fetus\;5\;is\;"genotype"|\;phase\;i)P(phase\;i)\\
& notice:\;\;\sum_{i=1}^i P_i  = 1\\
\;\;\\
&Therefore\;the\;risk\;for\;fetus\;5\;to\;be\;affect\;is:\\
\;\;\\
R &= P(fetus\;5\;is\;"ee"|\;observed\;phenotypes \;within\; the \;pedigree)\\
\;\\
& = {{P(ee,bc)}\over P(EE,bc)+P(Ee,bc)+P(ee,bc)}\\
\;\;\\
& = {{(1-\theta)^5\theta+(1-\theta)^4\theta^2+(1-\theta)^2\theta^4+(1-\theta)\theta^5}\over (1-\theta)^4+2(1-\theta)^2\theta^2+\theta^4}\\
&0 \leq R \leq 1
\end{aligned}$$










